components/camel-saxon/pom.xml

```diff
@@ -33,7 +33,7 @@
 
     <properties>
         <camel.osgi.import.additional>
-            net.sf.saxon.*;version="[9.9.0,10)"
+            net.sf.saxon.*;version="[9.9.0,11)"
```

Comment:
Those API changes later would mean that we require saxon 10, 11

Reply:
@davsclaus  can i keep only version 10
 net.sf.saxon.*;version="[10.0,10.3)

Reply:
It doesn't makes sense to restrict to 10.3 the range. Use 11 as upper limit

Reply:
Thanks oscerd. then i think there is no need to change

Reply:
I have already provided upper limit 11
net.sf.saxon.*;version="[9.9.0,11)"

Reply:
No it should be [10,11)

Reply:
+1, it needs to be strict [10,11)